### PR TITLE
Improve PR and issue templates

### PR DESCRIPTION
**Pull request by KOLANICH**
Please fill in the fields below to submit a pull request.  The more information
that is provided, the better.

Fixes: #255

## Description of the changes being introduced by the pull request:

Improved the template a bit.

## Please verify and check that the pull request fulfils the following requirements:

- [x] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

**Comment by joshuagl** [comment]
Thank you for submitting a change to securesystemslib. Your patch header is very sparse, so it's not clear to me precisely what is being fixed here. Will the HTML commented string be displayed when users open a new Pull Request?

**Comment by KOLANICH** [comment]
It will. As a comment within markdown. But it will be invisible in the issue. It is intended - the commented-out message is for the issue author.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
-Please fill in the fields below to submit a pull request.  The more information
-that is provided, the better.
+<!-- Please fill in the fields below to submit a pull request.  The more information
+that is provided, the better. -->
 
-**Fixes issue #**:
+<!-- Insert issue number here. See https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword for more info. -->
```
Comment (lukpueh) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: 👍  I like the html comment idea! It removes clutter from PRs when submitters forget to delete these lines.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+Fixes: #
 
-**Description of the changes being introduced by the pull request**:
+# Description of the changes being introduced by the pull request**:
```
[the remark this answers is not in the document]
Comment (KOLANICH) [reply .github/PULL_REQUEST_TEMPLATE.md]: Asterisks removed. Now h3 instead of h1.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,12 +1,14 @@
-Please fill in the fields below to submit a pull request.  The more information
-that is provided, the better.
+<!-- Please fill in the fields below to submit a pull request.  The more information
+that is provided, the better. -->
 
-**Fixes issue #**:
+<!-- Insert issue number here. See https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword for more info. -->
+Fixes: #
```
Comment (lukpueh) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Does "linking-a-pull-request-to-an-issue-using-a-keyword" work case-insensitively and if there's a colon in between?

Comment (lukpueh) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: NVM, looks like it does.

**Review by lukpueh — APPROVED** [review]
Thanks for the improvements, @KOLANICH!

For future reference, please follow our [commit message guidelines](https://github.com/secure-systems-lab/lab-guidelines/blob/master/commits.md) and consider spell-checking them.

**Pull request merged**